css/style.css

```diff
@@ -0,0 +1,706 @@
+  }
+}
+
+.picks img {
+  width: 28%;
+  height: 100%;
+}
+
+.picks p {
+  width: 60%;
+  font-size: 11px;
+  line-height: 12px;
+}
+
+.inflation-theory {
+  width: 60%;
+  height: 95vh;
+  margin: 0 auto;
+  display: grid;
+  grid-template: 100px repeat(4, 1fr) 100px / repeat(4, 1fr);
+  grid-template-areas: "header header header header" "figure1 figure2 figure3 figure4" "caption1 caption2 caption3 caption4" "figure5 figure6 figure7 figure8" "caption5 caption6 caption7 caption8" "footer footer footer footer";
+  grid-gap: 10px;
+}
+
+.header {
+  grid-area: header;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-size: 13px;
+}
+
+.caption {
+  font-size: 12px;
+  line-height: 15px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.inflation-theory img {
+  width: 100%;
+}
+
+.inflation-theory .img1 {
+  grid-area: figure1;
+}
+
+.inflation-theory .caption1 {
+  grid-area: caption1;
+}
+
+.inflation-theory .img2 {
+  grid-area: figure2;
+}
+
+.inflation-theory .caption2 {
+  grid-area: caption2;
+}
+
+.inflation-theory .img3 {
+  grid-area: figure3;
+}
+
+.inflation-theory .caption3 {
+  grid-area: caption3;
+}
+
+.inflation-theory .img4 {
+  grid-area: figure4;
+}
+
+.inflation-theory .caption4 {
+  grid-area: caption4;
+}
+
+.inflation-theory .img5 {
+  grid-area: figure5;
+}
+
+.inflation-theory .caption5 {
+  grid-area: caption5;
+}
+
+.inflation-theory .img6 {
+  grid-area: figure6;
+}
+
+.inflation-theory .caption6 {
+  grid-area: caption6;
+}
+
+.inflation-theory .img7 {
+  grid-area: figure7;
+}
+
+.inflation-theory .caption7 {
+  grid-area: caption7;
+}
+
+.inflation-theory .img8 {
+  grid-area: figure8;
+}
+
+.inflation-theory .caption8 {
+  grid-area: caption8;
+}
+
+.footer::before {
+  width: 60px;
+  content: '';
+  display: inline-block;
+  height: 1px;
+  margin: 0;
+  background-color: #000;
+}
+
+.footer {
+  grid-area: footer;
+}
+
+.telescope {
+  margin: 0;
+}
+
+.telescope img {
+  max-width: 100%;
+}
+
+.bottompart {
+  font-size: 0.75rem;
+  line-height: 1rem;
+  color: rgb(102, 102, 102);
+}
+
+.bottompart a {
+  color: rgb(102, 102, 102);
+}
+
+.moreinspace a {
+  color: black;
+  text-decoration: none;
+}
+
+.footer-links-logo a {
+  text-decoration: none;
+  color: #60839b;
+  font-size: 11px;
+}
+
+.popular-links a {
+  text-decoration: none;
+  color: black;
+  font-size: 1.1875rem;
+  line-height: 1.4375rem;
+}
+
+.footer-links-element a {
+  text-decoration: none;
+  color: #000;
+}
+
+.footer-contact-bar a {
+  text-decoration: none;
+  color: black;
+}
+
+.contacts li a {
+  text-decoration: none;
+  color: #666;
+}
+
+.footer-contact-bar {
+  border-top: 2px solid black;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: white;
+  height: 60px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 1;
+}
+
+.bottompart a:hover {
+  text-decoration: none;
+}
+
+.social-buttons {
+  display: flex;
+  justify-content: space-between;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_